unlock-app/package.json

```diff
@@ -103,7 +105,8 @@
     "start": "yarn build && NODE_ENV=production next start",
     "test": "UNLOCK_ENV=test jest --env=jsdom",
     "lint": "eslint --resolve-plugins-relative-to ../packages/eslint-config --ext .tsx,.ts,.js src/",
-    "ci": "yarn test && yarn lint"
+    "ci": "yarn test && yarn lint",
+    "e2e": "node ./__e2e__/index.js"
```

Comment:
Can we make sure `e2e` runs as part of CI though? Also I think we need ti use a runner... not `node`. Maybe you can replace `jest` with `vitest` first in another PR and then make sure the e2e tests are running fine.